src/main/java/meteordevelopment/meteorclient/systems/modules/misc/ServerSpoof.java

```diff
@@ -51,6 +51,14 @@ public class ServerSpoof extends Module {
         .build()
     );
 
+    private final Setting<Boolean> bogusAccept = sgGeneral.add(new BoolSetting.Builder()
+        .name("bogus-accept")
```

Comment:
Can you name it something better than `bogus accept`, maybe something along the lines of resource pack bypass

Reply:
I don't know about you but that is not more self explanatory to me

Reply:
hmm the function of this module is to send a "resource pack accepted" and shouldn't it be fake accept?

Reply:
Why not be honest and say it is to error check addresses.